app/services/UserService.py

```diff
@@ -19,15 +25,64 @@ async def modify_user(user_id: int, model: PydanticUserModify):
 
     if model.mail and await UserInDB.all().filter(mail=model.mail).count() != 0:
```

Comment:
You can refactor this line with the method you used in create_user.
UserInDB.filter(mail=model.mail).exists()